<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Training Schedule</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="addLayout">
      <item>
       <widget class="QLineEdit" name="entryEdit">
        <property name="placeholderText">
         <string>Enter training entry</string>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #333; border-radius: 5px; padding: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #333; border-radius: 5px; padding: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit">
        <property name="displayFormat">
         <string>hh:mm AP</string>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid #333; border-radius: 5px; padding: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #4CAF50; color: white; padding: 10px 20px; border: none; border-radius: 5px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="entriesList"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Delete Selected Entry</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #f44336; color: white; padding: 10px 20px; border: none; border-radius: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="text">
         <string>Logout</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #333; color: white; padding: 10px 20px; border: none; border-radius: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="feedbackButton">
        <property name="text">
         <string>Send Feedback</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #2196F3; color: white; padding: 10px 20px; border: none; border-radius: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="historyButton">
        <property name="text">
         <string>History</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #9C27B0; color: white; padding: 10px 20px; border: none; border-radius: 5px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="dateTimeLabel">
      <property name="text">
       <string>Current Date and Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
